Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN yum -y install wget gcc openssl-devel bzip2-devel libffi libffi-devel zlib-d
 RUN yum -y groupinstall "Development Tools"
 
 ##### Install PostgreSQL 10 client (psql)
-RUN yum -y install https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-7-x86_64/pgdg-redhat10-10-2.noarch.rpm
+RUN yum -y install https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
```

Comment:
Unfortunately this did not work for my docker-compose container build locally. Even though I can download the rpm file on my host. Thoughts?

![image](https://user-images.githubusercontent.com/22781949/81201359-6cacb600-8f82-11ea-882c-bb416c0a4936.png)

Reply:
I just tried `docker build --no-cache .` again and it built for me just fine. Maybe a network fluke? have you tried multiple times to build that layer or just the once?

Reply:
are you using any cached layers?

Reply:
must have been a VPN issue. Worked this afternoon